frontend/src/stories/testQueue/QueueItem.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Story, Meta } from "@storybook/react";
+
+import QueueItem, { QueueItemProps } from "../../app/testQueue/QueueItem";
+import { getMocks, StoryGraphQLProvider } from "../storyMocks";
+
+export default {
+  title: "App/Test Queue/Queue Item",
+  component: QueueItem,
+  parameters: {
+    layout: "fullscreen",
+    msw: getMocks(
+      "EditQueueItem",
+      "SubmitTestResults",
+      "GetPatientsLastResult",
+      "SendPatientLinkSms",
+      "UpdateAOE",
+      "RemovePatientFromQueue"
+    ),
+  },
+  decorators: [
```

Comment:
If your component needs graphql, make sure to add a decorator to the suite of stories or otherwise wrap the individual stories in the `StoryGraphQLProvider`